src/Sql/Sql/help/Clear-AzSqlServerVulnerabilityAssessmentSettings.md

```diff
@@ -0,0 +1,156 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/az.sql/clear-azsqlservervulnerabilityassessmentsettings
+schema: 2.0.0
+---
+
+# Clear-AzSqlServerVulnerabilityAssessmentSettings
+
+## SYNOPSIS
+Clears the vulnerability assessment settings of a server.
+
+## SYNTAX
+
+### ClearByNameParameterSet (Default)
+```
+Clear-AzSqlServerVulnerabilityAssessmentSettings [-ResourceGroupName] <String> [-ServerName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ClearByParentResourceParameterSet
+```
+Clear-AzSqlServerVulnerabilityAssessmentSettings [-InputObject] <AzureSqlServerModel>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Clear-AzSqlServerVulnerabilityAssessmentSettings** cmdlet removes the vulnerability assessment settings of an Azure SQL Server.
+Note that you need to run *Enable-AzSqlServerAdvancedThreatProtection* cmdlet as a prerequisite for using this cmdlet.
+
+## EXAMPLES
+
+### Example 1 - Clear the vulnerability assessment settings of an Azure SQL server
+```powershell
+PS C:\> Clear-AzSqlServerVulnerabilityAssessmentSettings `
+            -ResourceGroupName "ResourceGroup01" `
+            -ServerName "Server01"
+```
+
+### Example 2 - Clear the vulnerability assessment settings from a server object
+```powershell
+PS C:\> Get-AzSqlServer `
+            -ResourceGroupName "ResourceGroup01" `
+            -ServerName "Server01" `
+            | Clear-AzSqlServerVulnerabilityAssessmentSettings
+```
+
```

Comment:
Please add samples output.

Reply:
Clear-AzSqlServerVulnerabilityAssessmentSettings doesn't return anything on success

Reply:
the same comment.